org.eclipse.jdt.debug/model/org/eclipse/jdt/debug/core/IJavaBreakpoint.java

```diff
@@ -47,6 +47,12 @@ public interface IJavaBreakpoint extends IBreakpoint, ITriggerPoint {
 	 */
 	public static final int SUSPEND_THREAD = 2;
 
+	/**
+	 * Suspend policy constant indicating a breakpoint will not suspend the target VM when hit.
+	 *
+	 * @since 3.5
```

Comment:
expecting `@since 3.22`
```
00:01:32.006  [ERROR] Failed to execute goal org.eclipse.tycho:tycho-apitools-plugin:4.0.11:verify (verify) on project org.eclipse.jdt.debug: There are API errors:
00:01:32.006  [ERROR] model/org/eclipse/jdt/debug/core/IJavaBreakpoint.java:54 Invalid @since 3.5 tag on RESUME_THREAD; expecting @since 3.22
```

note that this  change has to wait for M1 anyway and then you may need to bump one more time